packages/serve-runtime/src/createServeRuntime.ts

```diff
@@ -104,9 +120,9 @@ export function createServeRuntime(config: MeshServeConfig) {
       setFetchFn(fetchAPI.fetch);
     },
     onYogaInit({ yoga }) {
-      const onFetchHooks: OnFetchHook<MeshServeContext>[] = [];
+      const onFetchHooks: OnFetchHook<YogaInitialContext & MeshServeContext>[] = [];
```

Comment:
Why `YogaInitialContext` is not already part of `MeshServerContext` ?

Reply:
Great insight! Its is now https://github.com/ardatan/graphql-mesh/pull/6748/commits/f15067e2d3f57b98d0b64dfec01e64e2c139e6ee.